portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.js

```diff
@@ -18,7 +18,8 @@ const DEFAULT_PAYMENT_OFFICER_TEAM_NAME = 'Team';
 const getReportDueDate = async () => {
   const bankHolidays = await externalApi.bankHolidays.getBankHolidayDatesForRegion(BANK_HOLIDAY_REGION.ENGLAND_AND_WALES);
   const businessDaysToAdd = process.env.UTILISATION_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH;
-  return addBusinessDaysWithHolidays(startOfMonth(new Date()), businessDaysToAdd, bankHolidays);
+  const businessDayIndex = businessDaysToAdd + 1;
```

Comment:
I think I'd expect this function to just call something that gives you bank the 10th business day, and not have to worry about adding/minusing anything